src/components/navbar/styles.ts

```diff
@@ -4,11 +4,9 @@ import { breakpoints, theme } from "~/styles/theme.js";
 
 export const styles = {
   container: {
-    position: "absolute",
-    top: 0,
-    left: 0,
-    right: 0,
+    position: "relative",
```

Comment:
this is a pretty impactful change as it will affect all routes, have you checked for sideffects? I've added padding in some pages to account for the navbar being absolute

Reply:
Yup! when use you a flex box with flex:0 (navbar) and flex:1 (content) there is no need for absoluting the navbar. The idea being the we are trying to avoid infinite scroll conditions of the parent component